### PR TITLE
Manure management

### PR DESCRIPTION
**Pull request by ltp23**
Main changes:
- `pen_manure.py`: Removed the manure- prefix in the name of most attributes.
- `gas_emissions.py`: Spelled out abbreviated parameter names. 
- `reception_pit.py` and `reception_pit_daily_output.py`: Reception pits are the second component in the overall process. 
- `manure_separator_classes.py` and `manure_separator_daily_output.py`: Separators are the third component. Usually, a separator receives the input from a reception pit, with one exception where the input can also come from the anaerobic digestion treatment. 

---

Preferred PR closing time: Friday, Dec 23

### CODE REVIEW

**Review by varrma — APPROVED** [review]

File: RUFAS/routines/manure/gas_emissions/gas_emissions.py [hunk 1]
```diff
@@ -138,17 +142,19 @@ def calc_ammonia_housing_emission(cls, num_animals: int,
...
 
         Args:
             num_animals: Number of animals in the pen.
             barn_area: surface area for treatment, m^2.
             manure_urine_total_ammoniacal_nitrogen: total ammoniacal nitrogen in manure urine, kg N.
             manure_urine: total amount of manure urine in exposed surface area, kg.
-            tempC: temperature, C.
-            hsc: housing specific constant, s/m.
+            temperature_celsius: temperature, C.
+            housing_specific_constant: housing specific constant, s/m.
```
Comment (ew3361zh) [inline RUFAS/routines/manure/gas_emissions/gas_emissions.py:157]: I'm not sure I understand what the `housing_specific_constant` is or does nor what the measurement units are. Could that be expounded upon a little further here? Or I see it's in reference to a constant in the GasEmissionsConstants class but didn't see a more expanded definition there.

File: RUFAS/routines/manure/pen_manure/pen_manure.py [hunk 2]
```diff
@@ -20,41 +20,41 @@ class PenManure:
     """A class that represents the manure data extracted from the animal module.
 
     Attributes:
-        manure_urea: manure_urea concentration in the manure, g/L.
-        manure_urine: amount of urine, kg.
-        manure_urine_ammoniacal_nitrogen: ammoniacal nitrogen concentration in the urine, g/L.
-        manure_total_ammoniacal_nitrogen: total ammoniacal nitrogen concentration in the manure slurry, g/L.
-        manure_nitrogen: nitrogen in liquid and solid manure, kg.
+        urea: manure urea concentration in the manure, g/L.
```
Comment (ew3361zh) [inline RUFAS/routines/manure/pen_manure/pen_manure.py:23]:
```suggestion
        urea: urea concentration in the manure, g/L.
```

**Review by ew3361zh — APPROVED** [review]
I think this looks great, Loi. I had a couple small suggested changes to wording and to explanations of what variables actually represent but otherwise this looks really good.

I ran it locally and everything looked good as well and all tests were passing.

File: RUFAS/routines/manure/manure_separators/manure_separator_classes.py [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+                total_daily_manure_volume=manure_separator_daily_input.liquid_manure_daily_volume,
+                final_solids_wet_mass=(
+                        manure_separator_daily_input.liquid_manure_total_solids *
+                        self.config.total_solids_removal_efficiency_for_separator /
+                        self.config.percent_dry_solids
+                ),
+
+                solid_manure_total_solids=(
+                        manure_separator_daily_input.liquid_manure_total_solids *
+                        self.config.total_solids_removal_efficiency_for_separator),
```
Comment (morrowcj) [inline RUFAS/routines/manure/manure_separators/manure_separator_classes.py:90]: For my own understanding: Why do you use parens in your assignments here (`x = (expression)`)? Is it simply break up the expression into multiple lines, or is it turning them into tuples for immutability?

**Review by morrowcj — APPROVED** [review]
I think this looks great. I really appreciate the work you put in to making the variable names more clear.

File: RUFAS/routines/manure/gas_emissions/gas_emissions.py (hunk 1 above)
Comment (morrowcj) [reply RUFAS/routines/manure/gas_emissions/gas_emissions.py:157]: I agree. I think the name is probably fine, but maybe add more info to the description.

File: RUFAS/routines/manure/manure_separators/manure_separator_classes.py (hunk 3 above)
Comment (ltp23) [reply RUFAS/routines/manure/manure_separators/manure_separator_classes.py:90]: Yes, this is only for multi-line formatting reasons. Sometimes I use `\` as well.

**Comment by ltp23** [comment]
Thank you everyone for reviewing the code. I will try to address some of the comments in the next PR.

**Pull request merged**